docs/docsite/rst/scenario_guides/guide_aws.rst

```diff
@@ -11,9 +11,9 @@ section is to explain how to put Ansible modules together (and use inventory scr
 
 Requirements for the AWS modules are minimal.
 
-All of the modules require and are tested against recent versions of boto.  You'll need this Python module installed on your control machine.  Boto can be installed from your OS distribution or python's "pip install boto".
+All of the modules require and are tested against recent versions of boto - check the module documentation for the minimum required version for each module.  You'll need the boto Python module installed on your control machine. Boto can be installed from your OS distribution or python's "pip install boto".
```

Comment:
Most of the AWS modules use boto3 instead of the original boto. The examples below use the `ec2` module which is one of those that need `boto`, but most AWS modules need `boto3`. Maybe the suggestion should be to install both - "pip install boto boto3".

Reply:
@s-hertel @jillr does ac0d0adfc8dfaf58291150035916ac30920c2706 make the paragraph better?

Reply:
Looks good to me!